### PR TITLE
Fix Bug(#32) IllegalArgumentException when SEND button click multiple times

### LINKED ISSUE #32: IllegalArgumentException when SEND button click 3 times
**Describe the bug**

App Stopped working due to FATAL EXCEPTION: main.

**To Reproduce**

Install App and allow Bluetooth, with no device connected. With empty field, click on SEND more than 3 times.

**Expected behavior**

If no device is connected, the device should give a message, may be in the form of Toast that device not connected or Scan could not find any device.

**Logs**

E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.nilhcem.blenamebadge, PID: 10565
    java.lang.IllegalArgumentException: scanner already started with given callback
        at no.nordicsemi.android.support.v18.scanner.BluetoothLeScannerImplLollipop.startScanInternal(BluetoothLeScannerImplLollipop.java:61)
        at no.nordicsemi.android.support.v18.scanner.BluetoothLeScannerCompat.startScan(BluetoothLeScannerCompat.java:112)
        at com.nilhcem.blenamebadge.device.bluetooth.ScanHelper.startLeScan(ScanHelper.kt:57)
        at com.nilhcem.blenamebadge.ui.message.MessagePresenter.sendBytes(MessagePresenter.kt:36)
        at com.nilhcem.blenamebadge.ui.message.MessagePresenter.sendBitmap(MessagePresenter.kt:25)
        at com.nilhcem.blenamebadge.ui.message.MessageActivity$onCreate$3.onClick(MessageActivity.kt:58)
        at android.view.View.performClick(View.java:6614)
        at android.view.View.performClickInternal(View.java:6587)
        at android.view.View.access$3100(View.java:784)
        at android.view.View$PerformClick.run(View.java:26125)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6715)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:911)
I/Process: Sending signal. PID: 10565 SIG: 9
Application terminated.

**Screenshots**

image

**Smartphone Info:**
|                            |                       |
|-------------------|----------------|
|Device                 |Nokia 5.1 Plus |
|Android Version  |Android Pie     |

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by itskshitizsh**
Fixes #32 

Changes: 
Add Try Catch for the exception handling with toast message. So that App doesn't stop working due to the exception arise.

### CODE REVIEW

File: app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt [hunk 1]
```diff
@@ -54,11 +54,14 @@ class MessageActivity : AppCompatActivity() {
 
         send.setOnClickListener {
             // Easter egg
+           try{
```
Comment (liveHarshit) [inline app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt]: Space after try.

File: app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt [hunk 2]
```diff
@@ -54,11 +54,14 @@ class MessageActivity : AppCompatActivity() {
...
             if (content.text.isEmpty()) {
                 presenter.sendBitmap(this, BitmapFactory.decodeResource(resources, R.drawable.mix2))
             } else {
                 presenter.sendMessage(this, convertToDeviceDataModel())
-            }
+            }}catch (e: Exception){
```
Comment (liveHarshit) [inline app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt]: Please follow the java style guideline. Or you can run `./gradlew spotlessApply` to fix it and build will pass.

**Comment by mariobehling** [comment]
@itskshitizsh Still up for it?

File: app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt [hunk 3]
```diff
@@ -54,11 +54,14 @@ class MessageActivity : AppCompatActivity() {
...
+               Toast.makeText(this,"Scan could not find any device",Toast.LENGTH_LONG).show()
```
Comment (CloudyPadmal) [inline app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt]: Also don't use hard-coded resources.

**Comment by mariobehling** [comment]
I am closing this due to inactivity. If you are interested to follow up, please reopen this PR. Thank you.

**Pull request closed without merging**